test/mkosi.default.networkd-test

```diff
@@ -57,6 +57,8 @@ BuildPackages=
         ninja-build
         pam-devel
         pcre2-devel
+        perl-IPC-SysV
+        perl-Time-HiRes
```

Comment:
`perl(IPV::SysV)` and `perl(Time::HiRes)` would be more canonical (resilient to package splits and merges).

We should convert most of those `-devel` deps to virtual provides too.